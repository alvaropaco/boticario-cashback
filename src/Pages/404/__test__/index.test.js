import React from 'react';
import NotFound from '../';
import renderer from 'react-test-renderer';
import { Router } from 'react-router-dom'
import { createBrowserHistory } from "history";

const history = createBrowserHistory();

it('renderiza a pÃ¡gina de Not Found', () => {
    const tree = renderer
    .create(
        <Router history={history}>
            <NotFound />
        </Router>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});