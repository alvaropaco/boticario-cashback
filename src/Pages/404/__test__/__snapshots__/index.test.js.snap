// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderiza a página de Not Found 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="404_website.svg"
    />
    <p>
      Paǵina não encontradada.
    </p>
    <a
      href="/"
      onClick={[Function]}
    >
      Voltar á Home
    </a>
  </header>
</div>
`;
